presto-server/src/main/assembly/presto.xml

```diff
@@ -184,5 +184,9 @@
             <directory>${project.build.directory}/dependency/presto-bigquery-${project.version}</directory>
             <outputDirectory>plugin/presto-bigquery</outputDirectory>
         </fileSet>
+        <fileSet>
+            <directory>${project.build.directory}/dependency/presto-iceberg-${project.version}</directory>
+            <outputDirectory>plugin/iceberg</outputDirectory>
```

Comment:
presto-iceberg or iceberg? either is fine, just a note

Reply:
I think it's fine. Just like mysql connector: [assembly/presto.xml#presto-mysql](https://github.com/prestodb/presto/blob/bb6d0935dc611bd3fe87963f571a4abe8370b117/presto-server/src/main/assembly/presto.xml#L120)